test/extend-string.test.js

```diff
@@ -0,0 +1,71 @@
+let string = require("../extend-string");
+let vowelString = 'seun';
+let consonantString = 'sxn';
+let capitalizedString = 'SEUN';
+let currencyDigits = '11111.11';
+let currency = '11,111.11';
+let name = 'Mr.Ben';
+let figureOfSpeech = 'onomatopoeia';
+let stringedNumbers = '325';
+let demoString = 'tyu';
+let singleDigit = '3';
+let doubleDigit = '34'
+let question = 'What is your name?';
+let question1 = 'is that your bag'
+
```

Comment:
consider grouping your tests into various describe blocks for specific methods. for example:
```javascript
//a describe block for testing a string.hasVowels method
describe("Extended String-methods", () => {
  describe("Test for hasVowel method", () => {
    it("Should return a true if a string contains a vowel", () => {
      expect(vowelString.hasVowels()).toBeTruthy();
    });
    //other relating tests for the hasVowel method goes here
  });
});


```